### PR TITLE
[In market test] Helper setup instructions

### PR DESCRIPTION
**Pull request by richajaindce**

### CODE REVIEW

**Comment by richajaindce** [comment]
@akoshelev let me know if instructions make sense

File: in-market-test/helper-nodes/setup_instructions.md [hunk 1]
```diff
@@ -6,14 +6,41 @@ Here are the instructions for setting up node for running IPA.
...
+   i. IP address/DNS which is accessible from public internet (i.e. firewall is not running)
+   ii. Confirm if machine is able to listen on port 80 
+
+2. Download docker image of IPA from shared folder. 
+    
+   We will be uploading an executable docker image which will have IPA code.
+   
+   Link: TBD
+
+3. Generate public and private keys for encrypting match keys and also, to secure the report files while in transit.
```
Comment (akoshelev) [inline in-market-test/helper-nodes/setup_instructions.md:29]: would it be better if docker image already have them?

File: in-market-test/helper-nodes/setup_instructions.md [hunk 2]
```diff
@@ -6,14 +6,41 @@ Here are the instructions for setting up node for running IPA.
...
    Command: `cargo build`
 5. Running a sample test : 
+   
    Command: `cargo bench --bench oneshot_ipa --features="enable-benches" --no-default-features`
    
-## Step 2: Run sample test across 3 local nodes
-   IPA team is still working on this and we will populate this section once the instructions are ready
+## Step 2: Setup helper configuration
+1. Provide details of how the machine can be accessed from public internet. For this, we would need to know
+   i. IP address/DNS which is accessible from public internet (i.e. firewall is not running)
```
Comment (akoshelev) [inline in-market-test/helper-nodes/setup_instructions.md:20]: s/firewall not running/firewall allows connecting to the helper port (443)

File: in-market-test/helper-nodes/setup_instructions.md [hunk 3]
```diff
@@ -6,14 +6,41 @@ Here are the instructions for setting up node for running IPA.
...
+   ii. Confirm if machine is able to listen on port 80
```
Comment (akoshelev) [inline in-market-test/helper-nodes/setup_instructions.md:21]: the standard port for HTTPS is 443 I think

File: in-market-test/helper-nodes/setup_instructions.md (hunk 1 above)
Comment (richajaindce) [reply in-market-test/helper-nodes/setup_instructions.md]: Havent added the instructions for this, will rely on you or @bmcase to add it.

**Review by akoshelev — APPROVED** [review]

**Pull request merged**